### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by xh3b4sd**
Fixes https://github.com/giantswarm/giantswarm/issues/4344.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -101,10 +101,6 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/randomkeys"
 
-[[constraint]]
-  branch = "master"
-  name = "github.com/giantswarm/randomkeytpr"
```
Comment (xh3b4sd) [inline Gopkg.toml]: This is now not used anymore. We use the equivalent `randomkeys` lib though.

**Comment by JosephSalisbury** [comment]
`+9,650 −107,974` <3

File: client/aws/aws.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/support/supportiface"
 	"github.com/giantswarm/microerror"
 
-	"github.com/giantswarm/aws-operator/service/controller/v2/resource/cloudformation/adapter"
+	"github.com/giantswarm/aws-operator/service/controller/v18/adapter"
```
Comment (xh3b4sd) [inline client/aws/aws.go]: This smells and should be refactored. We only import this for account ID validation, which is pretty basic. We do the validation in a couple of places though. Any idea how this should look like?

**Comment by xh3b4sd** [comment]
E2E tests are broken right now due to VPC limit issues which are addressed via another PM. I deployed this branch on `gauss` and that just seems to be fine. Merging.

**Comment by xh3b4sd** [comment]
Duh, I did not get an approval, only a comment. :D

File: client/aws/aws.go (hunk 2 above)
Comment (xh3b4sd) [reply client/aws/aws.go]: @giantswarm/sig-operator WDYT?

**Pull request merged**